Replaces all internal white space with one space

adkron opened the pull request:
This is to remove tags like "first \n name" and make them "first name"

adkron commented:
this was just supposed to be the top commit. Sorry, screwed up the pull request.

ches commented on .gitignore:
```diff
@@ -1,4 +1,4 @@
 rdoc
 Gemfile.lock
-
+.rvmrc
```
Please add `.rvmrc` to your global git `excludesfile`, it's a personal choice much like editors are.

ches commented:
The change looks good, but the history is confusing -- could you rebase your topic branch on current `integration` branch please?

adkron commented:
Sorry I screwed up the pull request. There are actually two things here because of that. If you want them both There is the replace all internal whitespace of tags with a single space, and allow a forced call to aggregate tags.

adkron commented:
I did a rebase and fixed a conflict. When I added the file it disappeared and wouldn't let me go on. So I just merged and fixed the conflict by hand.

adkron commented:
This should merge cleanly. I tried it locally. Sorry about all the confusion. I think I was working from and old branch.
